packages/expo-constants/src/Constants.server.ts

```diff
@@ -1,7 +1,7 @@
 import type { ExpoConfig } from 'expo/config';
-import type { Manifest as DevLauncherManifest } from 'expo-dev-launcher';
-import type { EmbeddedManifest, EASConfig, ExpoGoConfig } from 'expo-manifests';
+import type { EmbeddedManifest, EASConfig, ExpoGoConfig, ExpoUpdatesManifest } from 'expo-manifests';
```

Comment:
there were ci regression for this change: https://github.com/expo/expo/runs/44537338795

Reply:
Thanks for catching that @Kudo I'll get that fixed